src/components/MultiSelectGroup/MultiSelectGroup.stories.tsx

```diff
@@ -0,0 +1,317 @@
+import { VStack } from "@chakra-ui/react";
```

Comment:
Getting an error displaying the docs in Vercel: 
<img width="1298" alt="Screenshot 2024-04-02 at 1 23 00 PM" src="https://github.com/NYPL/nypl-design-system/assets/73774046/b4ed442c-e0f2-4a99-b25c-8d0f8d91327a">